client/index.ts

```diff
@@ -0,0 +1,18 @@
+import { provide } from 'angular2/core';
+import { bootstrap } from 'angular2/platform/browser';
+import { FORM_PROVIDERS } from 'angular2/common';
+import { HTTP_PROVIDERS } from 'angular2/http';
+import { App } from './components/index';
+import {
+  ROUTER_PROVIDERS,
+  LocationStrategy,
+  HashLocationStrategy,
+  APP_BASE_HREF
+} from 'angular2/router';
+
+bootstrap(App, [
```

Comment:
please use last RC5 of Angular 2 
At this version was added Module strategy:
@NgModule({
   bootstrap: (App)
})